### PR TITLE
changes image and change content as per my understading

### PR DESCRIPTION
**Pull request by makwanji**

### CODE REVIEW

File: about-us-2.html [hunk 1]
```diff
@@ -46,7 +46,7 @@
                     <div class="col-md-6 col-lg-6">
                         <ul class="top-info-links">
                             <li><i class="icofont-clock-time"></i> 7.30 AM - 9.30 PM</li>
-                            <li><i class="icofont-phone-circle"></i> +44 485854789</li>
```
Comment (makwanji) [inline about-us-2.html:49]: Add Country code

File: about-us-2.html [hunk 2]
```diff
@@ -72,7 +72,7 @@
         <nav class="navbar navbar-expand-lg navbar-light edu-navbar edu-navbar2 header-sticky">
             <div class="container">
                 <a class="navbar-brand" href="index.html">
-                    <img src="assets/img/logo-2.png" alt="Logo">
+                    <img src="assets/img/rudra .png" alt="Logo">
```
Comment (makwanji) [inline about-us-2.html]: plz remove space from filename.

File: assets/img/courses/1.jpg
Comment (makwanji) [inline assets/img/courses/1.jpg:1]: Image size is not correct.

Comment (tm4545) [reply assets/img/courses/1.jpg:1]: which image?

**Pull request closed without merging**